easybuild/easyblocks/p/python.py

```diff
@@ -31,7 +31,7 @@
 @author: Pieter De Baets (Ghent University)
 @author: Jens Timmerman (Ghent University)
 """
-
+import copy
```

Comment:
but but but but, we need something to bitch about! How can we consider this to be a true OSS project if we can't bitch about style?